tests/test_actions.py

```diff
@@ -52,3 +55,110 @@ def test_limbtest() -> None:
         obj.move_limbs(vel)
         assert obj.speed == vel
         vel += 3
+
+
+class Player(LimbControlMixin):
```

Comment:
why are you defining a player? Can't you just import it `from flatland.objects.items import Player` ?